src/drummap.jl

```diff
@@ -0,0 +1,54 @@
+
+# Map the pitches of midinotes to Instruments of Roland TD-50
+drummap = Dict{UInt8,String}(

```

Comment:
`drummap` should be declared as a constant. In addition a better name is `TD50_MAP` (all capitals on purpose) because the map is specific to TD50.